kumascript/src/api/wiki.ts

```diff
@@ -109,15 +109,6 @@ const wiki = {
     return path;
   },
 
-  // Build the URI of a given wiki page.
-  uri(path, query) {
```

Comment:
`wiki.uri` is still used in `kumascript/macros/minversionGeneric.ejs`, so those occurrences will need to be replaced.